lfroms/clouds
# Update Mapbox SDK to version 10

### What is the problem being solved in this PR?
This PR updates Clouds to use the newest version of the Mapbox SDK, version `10.0.0-rc.9` and updates the relevant documentation. The new version of the SDK is far more performant and uses the Metal rendering engine which is much lighter on CPU usage. It also loads map tiles faster and more reliably.

### Related issues or PRs
N/A

### Why did you choose this approach?
Keeping up to date with the latest libraries.

### How to test changes
1. Repeat this process for both ECCC and RainViewer radar sources.
1. Go to the Radar tab and play around with the map.
1. Use the playback controls.
1. Ensure that in all cases, the radar imagery is displayed.

### Checklist
